### ISSUE TITLE
Can't get this plugin to work on any vault other than your MathWiki

### ISSUE THREAD

**Issue by joelatschool** [zhaoshenzhai/obsidian-mathlinks#1]
Sorry to leave such a vague issue, but I've spent quite a bit of time trying to track down the discrepancy, and I haven't found it. I can't get this plugin to do anything, even on  a brand new vault -- no matter what I do all the links stay the same, no matter how many times I close and reopen the note, what viewing mode I switch to, or how many times I restart Obsidian. However, after downloading your [MathWiki](https://github.com/zhaoshenzhai/MathWiki), I can confirm that it does work for me there. I can also confirm that versions `<0.2.0` seem to work.

# Repro
1. Create a new vault.
2. Install `obsidian-mathlinks` version `>=0.2.0` into that vault, either through BRAT or through manually copying files.
3. Create these two pages:

`Test Page.md`:
```md
---
mathLink: $\sigma$
---

This page's link should display as $\sigma$.
```

`Test Page 2.md`:
```md
[[Test Page]]

The above link should display as $\sigma$.
```

After doing that, the result I get is that Test Page 2 displays like this:
<img width="785" alt="image" src="https://user-images.githubusercontent.com/94202132/195221701-93df4d86-b448-40a4-b045-69d82abb21f5.png">
no matter what I do.

**Comment by zhaoshenzhai** [comment]
Hello! Thanks for leaving an issue :)

The error seems to stem from the fact that I have wrongly set the `path` variable for each file as the default path that Obsidian uses to create your notes in (in my Mathwiki vault, that was `Notes`). For the test vault that you made, that path was `/` and so my plugin wrongly searched for a file called `//Test Page.md`.

The issue should be fixed now (in `0.2.3`), where the `path` variable now references the path that best matches the file name. This may still cause some issues (for example, if two notes with the same name are in different folders), which I will try to fix later and hopefully release in `0.2.4`.

**Comment by joelatschool** [comment]
`0.2.3` is working for me! Thanks so much!
